MAINTAINERS.md

```diff
@@ -86,7 +86,11 @@ $> git push
 
 ### 4. Push new version to pypi
 
-TODO: write this part
-
+```bash
+$> git checkout v42.8.0
```

Comment:
I think is better to push the new release from the master branch (less chance of making a mistake).